src/Jackett.Common/Definitions/uhdbits.yml

```
@@ -16,7 +16,10 @@ caps:
 
   modes:
     search: [q]
-    tv-search: [q, season, imdbid]
+    # special handling for season & episode:
+    # - no single episodes allowed on this tracker, so it's ignored
+    # - season search is mapped into keywords query
+    tv-search: [q, ep, season, imdbid]
```

```suggestion
    tv-search: [q, season, ep, imdbid]
```

This (or stripping both season and ep) is done for several other indexers to maintain compatibility with Sonarr, so we can remove the comment here. Then just change the order of `ep` and `season` to be consistent with other indexers.

---

happy to fix the order, but I'd prefer to keep the comment for extra clarity.